README.md

```diff
@@ -10,11 +10,17 @@ Repository structure:
 ├── output_API.schema.json
 ├── rna
 │   ├── rna.md
+│   ├── RIMA.png
```

Comment:
does this want to be moved to a `/imgs` like `chips.png`?

Reply:
done